### ISSUE TITLE
Add location relationships, ability to associate weather, soil, and other datatypes

### ISSUE THREAD

**Issue by bellerbrock** [solgenomics/sgn#2586]
Expected Behavior 
--------------------------------------------------------------------------
Most locations currently entered in the database track location at a general level, for example research station name. In reality most stations consist of many different fields. You might track weather data at just the station level, but track soil type, nutrient status, previous crop planted, etc for specific fields. 

Sweetpotato programs would like to more accurately reflect this in the database by specifying both station and exact field location within the station. For example Cunningham has over 50 specific fields, Clinton almost 200 [ClintonAndCunninghamMaps.pdf](https://github.com/solgenomics/sgn/files/3506749/ClintonAndCunninghamMaps.pdf). The fields could be represented graphically as described [here](https://github.com/solgenomics/sgn/issues/1570). Then weather data could be stored /linked to the station level and soil data at the field level

For Bugs:
---------

**Comment by nickmorales** [comment]
if we had a relationship table for the nd_geolocation table it would be easy to link, field locations within larger locations. Does the weather database use an nd_geolocation_id key to link between databases?

**Comment by bellerbrock** [comment]
no, just location name, but using nd_geolocation_id would be a good idea

**Comment by chris263** [comment]
Shall we close this @lukasmueller ? This is already available in trials detail page.
